cfgov/unprocessed/css/atoms/overlay.less

```diff
@@ -0,0 +1,37 @@
+/* topdoc
+  name: Overlay
+  family: cfgov-atoms
+  patterns:
+    - name: Default example
+      markup: |
+          <div class="a-overlay"></div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .a-overlay
+  tags:
+    - cfgov-atoms
+*/
+
+.a-overlay {
+    // Only show overlay at mobile/tablet size.
+    .respond-to-max( @bp-sm-max, {
+        height: 100%;
+        width: 100%;
+
+        position: fixed;
+        z-index: 10;
```

Comment:
Added TODO to move this value to a standardized value. Updated Slick carousel arrow value to 9 from 1000 in https://github.com/cfpb/cfgov-refresh/commit/5cfcf84a5c61150464c589181560325ff2335347